packages/perseus-editor/src/components/__stories__/color-select.stories.tsx

```diff
@@ -1,8 +1,9 @@
+import {type LockedFigureColor} from "@khanacademy/perseus";
```

Comment:
Can we move this back down to line 5 and revert it to 

```suggestion 
import type {LockedFigureColor} from "@khanacademy/perseus";
```